assets/headers/ivar_variants_header_mqc.txt

```diff
@@ -0,0 +1,8 @@
+#id: 'ivar_variants'
```

Comment:
Same question.
Not sure I understand why this file is here, or how it's used either.

Reply:
Similar to above file but this is a header file to create MultiQC custom content 😎
https://github.com/nf-core/viralrecon/blob/5ad5c00c81bb67c3eb32d87cf21c85b7d3c7e866/main.nf#L1372

The way it works that the counts/stats for each sample are appended to this header file and then collected at the end when passed to MultiQC to be added to the report.